Makefile

```diff
@@ -35,17 +34,17 @@ include $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/tools.mk
 
 .PHONY: install
 install:
-	@LD_FLAGS="-w $(shell EFFECTIVE_VERSION=$(EFFECTIVE_VERSION) $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh k8s.io/component-base $(REPO_ROOT)/VERSION $(EXTENSION_PREFIX)-$(NAME))" \
-	$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/install.sh ./...
+	@LD_FLAGS=$(LD_FLAGS) \
+	$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/install.sh ./cmd/...
```

Comment:
Why do we need this change?

Reply:
I exported `LD_FLAGS` as a variable so that it can also be used [here](https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/cd0ede12d77ff02d9c97e63bcd13c902da9e1b71/Makefile#L125)
In gardener we export it like [this](https://github.com/gardener/gardener/blob/b83a3a0932698ebf267462dff30f3e31c8f912ac/Makefile#L316)
It's not exported as a variable there, because I think we do not need it when running `make install` in g/g: [ref](https://github.com/gardener/gardener/blob/b83a3a0932698ebf267462dff30f3e31c8f912ac/Makefile#L89)

Reply:
I was rather asking about the `./cmd/...` change.